Calculator/index2.css

```diff
@@ -0,0 +1,44 @@
+
```

Comment:
rename this file into `style.css`
also fix the code style. http://codeguide.co/